README.md

```diff
@@ -53,7 +53,7 @@ $ forge test --fork-url "https://rpc.gnosis.gateway.fm" -vvvv
 ### Deploy
 
 ```shell
-$ forge script script/script/DeployYieldDisburser.s.sol:DeployYieldDisburser --rpc-url "https://rpc.gnosis.gateway.fm" --private-key <your_private_key> --verify --etherscan-api-key <gnosis_scan_key> --chain 100
+forge script script/deploy/DeployYieldDisburser.s.sol:DeployYieldDisburser --rpc-url "https://rpc.gnosis.gateway.fm"   --broadcast --private-key <pk> --verify --etherscan-api-key  <gnosisscan_api_key>  --verifier etherscan --verifier-url "https://api.gnosisscan.io/api\?"
```

Comment:
should `"https://api.gnosisscan.io/api\?"` just be `"https://api.gnosisscan.io/api"`

Reply:
I was going by what this dev wrote https://github.com/foundry-rs/foundry/issues/5656#issuecomment-1684030163 

But honestly the only way I've managed to actually verify on gnosisscan is with the standard json input so I'll be removing this

Reply:
resolved with 23f47a3